db/test/versions/0015_test.js

```diff
@@ -0,0 +1,54 @@
+const _ = require('lodash');
+const helper = require('../helper');
+const testing = require('taskcluster-lib-testing');
+const Entity = require('taskcluster-lib-entities');
+
+const THIS_VERSION = 15;
```

Comment:
Neat trick to get the version number from the file name: https://github.com/taskcluster/taskcluster/pull/3108

Reply:
Thanks, I've added it.